contents/data_vis_makie_attributes.md

```diff
@@ -80,6 +80,14 @@ sco(s)
 Other positions are also available by combining `left(l), center(c), right(r)` and `bottom(b), center(c), top(t)`.
 For instance, for left top, use `:lt`.
 
+<!--
+When I remember correct (I didn't check it now), up to now you really just have
+put stuff in backticks that was code. Now it seem that it is also used to
+emphasize stuff like `legend` or `left(l), center(c), right(r)` and `bottom(b),
+center(c), top(t)`. If it is really just emphasizing and not *also* code, then
```

Comment:
yes, only these are like this, but because later they used as keyword arguments. But maybe is better to use a bold face font instead.

Reply:
As I stated in the next sentence, I'd prefer italics. I think you have used that in other places as well. When I also remember you used bold for highlighting *statements* (that one should remember).